### DISCUSSION TITLE
Adding new agents not working

### DISCUSSION THREAD

**Discussion by inovve** [wazuh/wazuh#17989] [General]
[![enter image description here][1]][1]On a fresh Ubuntu VPS, fresh install using docker, when trying to add a new agent, instead of "Close" option of that screen I get "Refresh" option. I then fill all the data, run the commands on the machines where agents suppose to run, nothing happens, if I press the refresh button it clears all options, if I go back to agents the list is empty.

On agent machine I get this in the logs:
wazuh-agentd: ERROR: (1208): Unable to connect to enrollment service at '[37.120.179.10]:1515'

**Comment by mdiego92** [comment]
Hello @inovve,

The error message you shared is because the agent cannot connect to the port 1515 TCP of your manager (37.120.179.10).
Could you please try executing `Netcat` or `Telnet` from your agent to verify if it's able to reach your manager?

`telnet 37.120.179.10 1515`
or
`nc -v 37.120.179.10 1515
`

By default, Wazuh Manager needs ports 1515 and 1514 TCP for the agents to be able to enroll and connect, respectively.
Documentation for reference:  https://documentation.wazuh.com/current/getting-started/architecture.html#required-ports

Looking forward to your response!
